src/views/preview/preview.jsx

```diff
@@ -315,7 +333,6 @@ class Preview extends React.Component {
                         replies={this.props.replies}
                         reportOpen={this.state.reportOpen}
                         studios={this.props.studios}
-                        user={this.props.user}
```

Comment:
I don't see why this is being removed, is this cruft?

Reply:
Yes, it was cruft, unused in preview presentation.